cartography/intel/aws/ec2.py

```diff
@@ -249,6 +254,7 @@ def load_ec2_security_groupinfo(session, data, region, current_aws_account_id, a
             GroupId=group_id,
             GroupName=group.get("GroupName", ""),
             Description=group.get("Description", ""),
+            VpcId=group.get("VpcId", ""),
```

Comment:
Recommend replacing `""` with `None` if the member does not exist.  In my opinion it makes more sense if the property does not exist for it to not be on the node at all than as an empty string.